homework11_gen.py

```diff
@@ -0,0 +1,41 @@
+"""
+Homework11.
+
+Description:
+    write generator function that has as input some range value and chunk_size
+    produces output as lists with len == chunk size
+    Example:
+    Call: chunk_generator(range(11), chunk_size=3)
+    Output:
+    [0, 1, 2]
+    [3, 4, 5]
+    [6, 7, 8]
+    [9, 10]
+"""
+
+
+def chunk_generator(range_value, chunk_size):
+    """
+    Generate chunked list according to entered range value and chunk size.
+
+    Args:
+        range_value (int):  Limit for the list to be chunked
+        chunk_size (int): The chunk size
+
+    Yields:
+        sliced_list[el:el+chunk_size]: sliced list with [start:end] positions
+
+    Returns:
+        None
+    """
+    sliced_list = list(range(range_value))
```

Comment:
list is resource consuming operation especially for great numbers. 
Range allows to do slices. Please examine example below
```
def chunks_generator(range_val, chunk_size=3):
    _range = range_val
    while _range:
        yield list(_range[:chunk_size])
        _range = _range[chunk_size:]


print(*[chunk for chunk in chunks_generator(range(23), chunk_size=5)], sep='\n')
```
Main diff -> list generation only for chunk